mkosi

```diff
@@ -1316,6 +1431,7 @@ def parse_args():
                        help='Make image bootable on EFI (only raw_gpt, raw_btrfs, raw_squashfs)')
     group.add_argument("--read-only", action='store_true', help='Make root volume read-only (only raw_btrfs, subvolume)')
     group.add_argument("--verity", action='store_true', help='Add integrity partition (implies --read-only)')
+    group.add_argument("--luks-encryption", choices=("all", "data"), help='Encrypt evrything except: esp (all) or esp and root (data)')
```

Comment:
I think I'd just call this option `--encryption=`. It's sufficient, and simpler I think. And LUKS is an implementaiton detail I think, which deserves being mentioned in the description, but not in the option name, I think